### ISSUE TITLE
BEGIN and END instructions not working in htmlvis

### ISSUE THREAD

**Issue by amir-zeldes** [korpling/ANNIS#474]
Null pointer exception on get annotation name for pseudo matcher.

**Comment by thomaskrause** [comment]
I reproduced this error with the following configuration file on the pcc2 corpus:

```
tok	span; style="tok"	value
annis:BEGIN span	"Hello World"
```

The NullPointerException occurs at
https://github.com/korpling/ANNIS/blob/develop/annis-visualizers/src/main/java/annis/visualizers/htmlvis/HTMLVis.java#L454

I at first thought this might be a regression but I can't find a version where this construct works. I have some trouble understand the logic behind of some of the code where the instruction priorities are collected and from my understanding this has always been problematic when "annis:BEGIN"/"annis:END" with a constant value are used.

**Comment by thomaskrause** [comment]
git bisecting revealed that f06e37cddd329bc52a48f62a50da91cd9af5a194 was the commit introducing this bug

```
thomas@ling144-121:~/annis$ git bisect good
f06e37cddd329bc52a48f62a50da91cd9af5a194 is the first bad commit
commit f06e37cddd329bc52a48f62a50da91cd9af5a194
Author: amir-zeldes <az364@LING-R9011CKZ.georgetown.mei.georgetown.edu>
Date:   Thu Sep 18 23:23:09 2014 -0400

    HTML visualizer now prioritizes element hierarchy for annotations covering the same tokens accordings to the order of the config file, not alphabetically (fix for issue #333)

:040000 040000 db03c1dabc0a82e699f62c2a0c1ac5881d65e9f6 e590bb4ac5ce660ec72bcf93c57005b2b6b29c02 M	annis-visualizers
```

**Comment by amir-zeldes** [comment]
Yes, I thought that was it since I didn't remember the 'priorities' logic
from my pseudoregion implementation. I'm testing a fix and should have a
pull request for you in a moment.

sent from my mobile
